### PR TITLE
filled firstName and lastName fields from user data

### PR DESCRIPTION
**Pull request by Tolik170**
Filled firstName and lastName fields from user data

### CODE REVIEW

**Review by Yurenko — APPROVED** [review]

File: tests/unit/pages/tutor-home/TutorHome.spec.jsx [hunk 1]
```diff
@@ -14,18 +22,19 @@ const TutorHomeWithProviders = () => (
 )
 
 describe('TutorHome component', () => {
+  mockAxiosClient.onGet(`${URLs.users.get}/${userId}`).reply(200, { data: userDataMock })
+
   const firstLoginState = {
-    appMain: { isFirstLogin: true, userRole: 'tutor' }
+    appMain: { isFirstLogin: true, userRole: 'tutor', userId: userId }
```
Comment (romanortynskyi) [inline tests/unit/pages/tutor-home/TutorHome.spec.jsx:28]: Replace userId: userId with userId.

File: tests/unit/pages/tutor-home/TutorHome.spec.jsx [hunk 2]
```diff
@@ -14,18 +22,19 @@ const TutorHomeWithProviders = () => (
...
   }
   const secondLoginState = {
-    appMain: { isFirstLogin: false, userRole: 'tutor' }
+    appMain: { isFirstLogin: false, userRole: 'tutor', userId: userId }
```
Comment (romanortynskyi) [inline tests/unit/pages/tutor-home/TutorHome.spec.jsx:31]: Same here.

File: tests/unit/pages/tutor-home/TutorHome.spec.jsx (hunk 1 above)
Comment (Tolik170) [reply tests/unit/pages/tutor-home/TutorHome.spec.jsx]: fixed

File: tests/unit/pages/tutor-home/TutorHome.spec.jsx (hunk 2 above)
Comment (Tolik170) [reply tests/unit/pages/tutor-home/TutorHome.spec.jsx]: fixed

**Review by KhrystynaPavlikovska — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**